python/plugins/processing/algs/grass7/Grass7Algorithm.py

```diff
@@ -788,16 +808,19 @@ def loadVectorLayer(self, name, layer, external=False):
             destFilename)
         self.commands.append(command)
 
-    def exportVectorLayerFromParameter(self, name, parameters, context):
+    def exportVectorLayerFromParameter(self, name, parameters, context, layer=None, nocats=False):
         """
-        Creates a dedicated command to export a raster from
-        temporary GRASS DB into a file via gdal.
-        :param grassName: name of the parameter
-        :param fileName: file path of raster layer
-        :param colorTable: preserve color Table.
+        Creates a dedicated command to export a vector from
+        a QgsProcessingParameter.
+        :param name: name of the parameter.
+        :param context: parameters context.
+        :param layer: for vector with multiples layers,exports only one layer.
```

Comment:
s/multiples/multiple
missing spacing after the comma
For consistency with other descriptions (see eg L845), should it be "exports" or "export"?